app/change_sets/coin_change_set.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+class CoinChangeSet < ChangeSet
+  delegate :human_readable_type, to: :model
+  apply_workflow(DraftCompleteWorkflow)
+
+  include VisibilityProperty
+  property :department, multiple: false, required: false
+  property :size, multiple: false, required: false
+  property :die_axis, multiple: false, required: false
+  property :weight, multiple: false, required: false
+  property :donor, multiple: false, required: false
+  property :deposit_of, multiple: false, required: false
+  property :seller, multiple: false, required: false
```

Comment:
Like donor, there doesn't seem to be a single field for information about the seller. The Accessions table `Info` field is as close as I can find.